### ISSUE TITLE
Remove iron-icons dependency

### ISSUE THREAD

**Issue by rubenstolk** [bendavis78/paper-chip#15]
It's not a good pattern to include all `iron-icons` when only one icon is needed.

**Comment by admwx7** [comment]
I agree @rubenstolk and will move iron-icons to being a optional dependency as part of the 2.0 release.

Once the Polymer team completes their tooling for resolving duplicated icons we might be able to move to a more elegant approach, in the meantime we'll just have to make due.
